README.md

```diff
@@ -1,101 +1,137 @@
-# @looksrare/sdk-v2
+# @hypercerts-org/marketplace-sdk
 
-![GitHub package.json version](https://img.shields.io/github/package-json/v/LooksRare/sdk-v2) ![GitHub](https://img.shields.io/github/license/LooksRare/sdk-v2) ![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/LooksRare/sdk-v2/build.yml) ![npm](https://img.shields.io/npm/dt/@looksrare/sdk-v2)
+![GitHub package.json version](https://img.shields.io/github/package-json/v/hypercerts-org/marketplace-sdk) ![GitHub](https://img.shields.io/github/license/hypercerts-org/marketplace-sdk) ![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/hypercerts-org/marketplace-sdk/build.yml) ![npm](https://img.shields.io/npm/dt/@hypercerts-org/marketplace-sdk)
 
-A collection of typescript tools to interact with LooksRare protocol V2 smart contracts 👀💎
+A collection of TypeScript tools to interact with HypercertsExchange smart contracts, enabling users to sell and buy Hypercerts on the marketplace.
 
-## Install
 
-This package has a peer dependency on [etherjs V5](https://docs.ethers.io/v5/).
+## SDK usage
 
-```bash
-yarn add @looksrare/sdk-v2 ethers
-```
+<!-- TODO: Check if links still work on deployment -->
+Read the [guides](./guides) if you need help with the implementation.
```

Comment:
These will reference to the hypercerts.org/docs, right?

Reply:
Yes, will update this when moving the guides to @hypercerts-org/docs.